ui/src/components/overview/MobileClientCardViewItem.js

```diff
@@ -30,42 +31,47 @@ const MobileClientCardViewItem = props => {
   } = props;
   return (
     <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3">
-      <Card matchHeight /* accented */ className="mobile-client-card">
-        <CardHeading>
-          <DropdownKebab id={app.metadata.name} pullRight className="card-dropdown-kebab">
-            <EditItemButton item={app} />
-            <DeleteItemButton itemType="app" itemName={appName} item={app} />
-          </DropdownKebab>
-          <CardTitle>
-            {buildTabEnabled && builds.numFailedBuilds > 0 ? (
-              <span>
-                <span className="pficon pficon-error-circle-o" />
-                {builds.numFailedBuilds}
-              </span>
-            ) : null}
-            {buildTabEnabled && builds.numInProgressBuilds > 0 ? (
-              <span>
-                <span className="pficon fa fa-refresh fa-spin fa-fw" />
-                {builds.numInProgressBuilds}
-              </span>
-            ) : null}
-            {!buildTabEnabled || (builds.numFailedBuilds === 0 && builds.numInProgressBuilds === 0) ? <span /> : null}
-          </CardTitle>
-        </CardHeading>
+      <Link to={`/mobileclient/${app.metadata.name}`}>
+        <Card matchHeight /* accented */ className="mobile-client-card">
+          <CardHeading>
+            <DropdownKebab id={app.metadata.name} pullRight className="card-dropdown-kebab">
+              <EditItemButton item={app} />
+              <DeleteItemButton itemType="app" itemName={appName} item={app} />
+            </DropdownKebab>
+            <div className="card-pf-title">
+              <h1>{app.spec.name}</h1>
+            </div>
+
+            <CardTitle>
+              {buildTabEnabled && builds.numFailedBuilds > 0 ? (
+                <span>
+                  <span className="pficon pficon-error-circle-o" />
+                  {builds.numFailedBuilds}
+                </span>
+              ) : null}
+              {buildTabEnabled && builds.numInProgressBuilds > 0 ? (
+                <span>
+                  <span className="pficon fa fa-refresh fa-spin fa-fw" />
+                  {builds.numInProgressBuilds}
+                </span>
+              ) : null}
+              {!buildTabEnabled || (builds.numFailedBuilds === 0 && builds.numInProgressBuilds === 0) ? <span /> : null}
+            </CardTitle>
```

Comment:
According to the flats, the builds info should be moved to the footer. It's ok if we don't implement it for now, let's comment this one out, and add a todo here.

Reply:
Moved the build info to the footer